build.gradle

```diff
@@ -26,7 +26,7 @@ def jacksonVersion = '2.10.2'
 gradleLint.rules = ['unused-dependency']
 
 dependencies {
-    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.1.6'
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '0.3.3'
```

Comment:
this should have been its own branch, create a PR for this, and then branch out from that branch to continue your work.